vcl_templates/www.vcl.erb

```diff
@@ -357,6 +357,12 @@ sub vcl_fetch {
 
   set beresp.http.Fastly-Backend-Name = req.http.Fastly-Backend-Name;
 
+  # Fastly doesn't recognise 307 as cacheable by default as it is based on an
+  # old version of Varnish that also lacked 307 support.
+  if (beresp.status == 307) {
+    set beresp.cacheable = true;
```

Comment:
Perhaps we should also check `beresp.http.Cache-Control` to check if the response should be cached, just in case. What do you think?

We could either add checks like `&& beresp.http.Cache-Control !~ "private" && ...` to the conditional or move this block down to line 417.

Reply:
Thanks Bill. I don't think we need to check the cache control headers as there are already conditionals for that which will result in a pass.

Moving this code to be below those checks, as you suggest, sounds like a good idea though to resolve some confusion about whether `beresp.cacheable` can impact those pass responses.

Reply:
Ah thanks for clarifying that :)